EmpleoDotNet/Views/Home/Index.cshtml

```diff
@@ -1,10 +1,35 @@
 ﻿@using EmpleoDotNet.ViewModel
 @model List<EmpleoDotNet.Core.Domain.JobOpportunity>
-    @{
+
+@{
     ViewBag.Title = "";
-    }
+}
+
+@* This added here because the partial view _FindsJobPartinal need this JS to work :( *@
+@section scripts {
+    <script type="text/javascript" src="http://maps.googleapis.com/maps/api/js?libraries=places"></script>
+    <script type="text/javascript">
+
+        google.maps.event.addDomListener(window, 'load', function () {
+            var places = new google.maps.places.Autocomplete(document.getElementById('location-input'));
+            google.maps.event.addListener(places, 'place_changed', function () {
+                var place = places.getPlace();
+                if (place) {
+                    var placeId = place.place_id;
+
+                    $('#location-placeid').val(placeId);
+                } else {
+                    $('#location-input').val('');
+                    $('#location-placeid').val('');
+                }
+            });
+        });
+    </script>
+}
+
+
+@Html.Partial("_FindJobsPartial", (JobOpportunitySearchViewModel)ViewBag.SearchViewModel)
 
```

Comment:
I see what you mean. We have tree afected files:

1. Index.cshtml
2. Detail.cshtml
3. New.cshtml

Both index and detail are the same, but new is different. We can create a new issue for this to organize this code into a single javascript object.